### PR TITLE
Convert to gomodules and upgrade to 1.19

### PR DESCRIPTION
**Pull request by jason-salesforce**
I have no real opinions on the actual structure of the code aside from the actual conversion to modules, if you do let me know and I'm happy to restructure it to whatever.

### CODE REVIEW

File: Procfile [hunk 1]
```diff
@@ -1 +1 @@
-web: urinception-web
+web: go run main.go
```
Comment (jabrown85) [inline Procfile]: Will this work? I assumed the go buildpack would not put the go tooling into the final slug

Comment (jason-salesforce) [reply Procfile]: Hmm that sounds right.

Comment (jason-salesforce) [reply Procfile]: https://github.com/heroku/urinception/pull/11/commits/2174bbb723b750e37bb1cc16b467f5bba77aa4d6

**Review by jabrown85 — APPROVED** [review]
This seems to work for me. I stole an example uri from a direwolf test and my staging cloud app returned the expected tarball.

Regardless - Once deployed, keep an eye out for direwolf test failures and be ready to revert.

We might also go ahead and set the stack to `heroku-22` prior to merging this.

**Pull request merged**